.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+*/**/node_modules
```

Comment:
not sure, but why not use the exact path that you know its going to be at? No need to play with wildcards in this instance.